packages/sdk-keysigner/index.ts

```diff
@@ -78,6 +78,62 @@ export class BrowserKeySigner implements CredentialSigner<KeyAssertion> {
     ])
   }
 
+  toHex = (buffer: ArrayBuffer): string => {
+    const view = new Uint8Array(buffer)
+    let hexString = ""
+    for (const byte of view) {
+      const hexByte = byte.toString(16)
+      hexString += hexByte.padStart(2, "0")
+    }
+    return hexString.toLowerCase()
+  }
+
+  exportPublicKeyInPemFormat = async (key: CryptoKeyPair): Promise<string> => {
+    const exported = await crypto.subtle.exportKey('spki', key.publicKey)
+    const pem = `-----BEGIN PUBLIC KEY-----\n${toBase64(exported)}\n-----END PUBLIC KEY-----`
```

Comment:
I believe, we also need to break the base64 encoding with new line every 64 characters to follow strict PEM formatting.

Reply:
I put the function in utils and split to 64 bytes per line. Feel free to resolve the conversation if that's ok for you.

Reply:
Closing this. Feel free to reopen it if you have an additional comment.